apps/codeforafrica/src/lib/api.netlify-cms/getSeo.js

```diff
@@ -0,0 +1,15 @@
+import { join } from "path";
+
+import getSettings from "./sections/getSettings";
+import { getCollectionBySlug } from "./utils";
+
+const indexPageDir = join(process.cwd(), "content/pages");
+
+export default function getSeo(page, fields = ["seo"]) {
+  const generalSettings = getSettings("general");
+  const defaultSeo = generalSettings?.seo;
+
+  const { seo } = getCollectionBySlug(indexPageDir, page, fields).items;
+
+  return Object.assign(defaultSeo, seo);
```

Comment:
You need deep merge here. `import { deepmerge } from '@mui/utils';` at the top and
```suggestion
  return deepmerge(defaultSeo, seo, { clone: true }); 
```